### ISSUE TITLE
The division of the LLFF dataset

### ISSUE THREAD

**Issue by Supermonsterli** [hanyangyu1021/LMGaussian#16]
Hello, may I ask how to divide the training set and test set in the experiment of 3-view input of LLFF dataset? Can you provide a partitioned LLFF dataset? Thank you very much!
